src/python/WMCore/REST/Main.py

```diff
@@ -387,41 +387,48 @@ def kill_daemon(self, silent=False):
 
     def start_daemon(self):
         """Start the deamon."""
+        # REST can be run in the frontend i.e. interactively e.g. to use pdb
+        # by setting the env. var.: DONT_DAEMONIZE_REST=True
+        # if the variable is missing or se to any other value, start noramlly as a daemon
```

Comment:
can you please fix a few typos in this line?
And in the line below, add a missing space that has been likely reported by jenkins, thus something like:
```
daemonize = os.getenv('DONT_DAEMONIZE_REST', 'False') != 'True'
```